src/common/config_opts.h

```diff
@@ -530,7 +530,7 @@ OPTION(mds_inject_traceless_reply_probability, OPT_DOUBLE, 0) /* percentage
 OPTION(mds_wipe_sessions, OPT_BOOL, 0)
 OPTION(mds_wipe_ino_prealloc, OPT_BOOL, 0)
 OPTION(mds_skip_ino, OPT_INT, 0)
-OPTION(max_mds, OPT_INT, 1)
+OPTION(max_mds, OPT_INT, 1) // Obsolete and ignored! Use `ceph fs set max_mds` instead.
```

Comment:
Why not just kill it entirely? I don't think unknown config options will break anything, and leaving it in is confusing.